### PR TITLE
Logged token is remove from CH. Added junit tests.

### PR DESCRIPTION
**Pull request by vojkanBukumiric91**

### CODE REVIEW

File: src/test/java/it/eng/idsa/businesslogic/service/impl/ClearingHouseServiceImplTest.java [hunk 1]
```diff
@@ -49,10 +52,12 @@
 	public void init() {
 		MockitoAnnotations.initMocks(this);
 		message = TestUtilMessageService.getArtifactResponseMessage();
```
Comment (IgorBalog-Eng) [inline src/test/java/it/eng/idsa/businesslogic/service/impl/ClearingHouseServiceImplTest.java:54]: Why making 2 calls, one without token and another with token, and storing result in same variable?

File: src/test/java/it/eng/idsa/businesslogic/service/impl/MultipartMessageServiceImplTest.java [hunk 2]
```diff
@@ -117,4 +117,15 @@ public void getTokenTest() throws JsonProcessingException {
 		String token = service.getToken(TestUtilMessageService.getArtifactRequestMessageWithToken());
 		assertNotNull(token);
 	}
+	@Test
+	public void removeTokenFromMessageTest() {
```
Comment (IgorBalog-Eng) [inline src/test/java/it/eng/idsa/businesslogic/service/impl/MultipartMessageServiceImplTest.java:121]: Try to be consistent, either all test methods ends with Test or none of them :)

File: src/test/java/it/eng/idsa/businesslogic/service/impl/ClearingHouseServiceImplTest.java (hunk 1 above)
Comment (vojkanBukumiric91) [reply src/test/java/it/eng/idsa/businesslogic/service/impl/ClearingHouseServiceImplTest.java:54]: I will fix that

Comment (IgorBalog-Eng) [reply src/test/java/it/eng/idsa/businesslogic/service/impl/ClearingHouseServiceImplTest.java]: Fix ok :)

**Pull request merged**